src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,26 @@
+import clsx from 'clsx';
+import { ComponentProps } from 'react';
+
+interface Props<T> {
+  value: T;
+  type: ComponentProps<'input'>['type'];
+  onChange: ComponentProps<'input'>['onChange'];
+}
```

Comment:
`ComponentProps` 타입이 중복되어서 사용되고 있네요!

```suggestion
interface Props<T> extends Omit<ComponentProps<'input'>, 'value'> {
  value: T;
}
```

이렇게는 어떨까요?

Reply:
위의 HFlex와 같이 type과 onChange 프로퍼티는 필수적으로 받아야 하는 값으로 정의하고 싶었는데, 이런 정의가 부적절하다고 생각하실까요?
만약 type과 onChange 프로퍼티를 필수적으로 받는게 적절할 수 있다면, ComponentProps를 중복해서 사용해야할 것 같은데, 다른 좋은 방법이 있을까요?